addons/mail/static/src/core/common/thread_model.js

```diff
@@ -264,22 +233,6 @@ export class Thread extends Record {
     canPostOnReadonly;
     /** @type {Boolean} */
     is_editable;
-    /**
-     * This field is used for channels only.
-     * false means using the custom_notifications from user settings.
-     *
-     * @type {false|"all"|"mentions"|"no_notif"}
-     */
-    custom_notifications = false;
-    mute_until_dt = fields.Datetime();
```

Comment:
Theorically, fields should always be defined in common bundle. Image you start the LC on a page, livechat bundle is loaded. Then, we start the portal, loading the portal bundle. Portal bundle defines another field, but model manager is already started => field is not processed as it should.